WebRoot/js/zimbraMail/share/model/ZmSettings.js

```diff
@@ -769,7 +769,6 @@ function(list, callback, result) {
 	if (resp.ModifyPrefsResponse != null) {
 		// notify each changed setting's listeners
 		for (var i = 0; i < list.length; i++) {
-			var setting = list[i];
```

Comment:
Doesn't this removal cause an error at the next line as `setting` is `undefined`.
```
undefined.origValue = undefined.copyValue();
```

Reply:
Removed it by mistakenly. Revert the changes.